app/editor/src/components/form/wysiwyg/Wysiwyg.tsx

```diff
@@ -117,29 +123,34 @@ export const Wysiwyg: React.FC<IWysiwygProps> = ({ fieldName, label, required })
         onClickRemoveFormat={stripHtml}
         onClickFormatRaw={onClickFormatRaw}
         onClickClear={() => setState({ ...state, html: '', rawHtml: '' })}
+        innerRef={setToolBarNode}
```

Comment:
I really don't understand this situation of using react state for a ref?

Reply:
It's kind of an odd instance, and a limitation of react-quill and the way we want to use it. But pretty much this will set the toolbar container to a unique instance of a `toolBarNode`.  Which can be found on line 94:
`    const config = {
      toolbar: {
        container: toolBarNode,
      },
    };`

Reply:
Experimenting with forward ref and looking for other solutions still, yet to find one

Reply:
Tried passing as setState instead of a ref - but no luck. Think this is one of those instances where we are fighting the library 
:(

Reply:
But the above is supported by React `Instead of passing a ref attribute created by createRef(), you pass a function. The function receives the React component instance or HTML DOM element as its argument, which can be stored and accessed elsewhere.` - from the docs